### ISSUE TITLE
Fix npm install warnings

### ISSUE THREAD

**Issue by feross** [feross/webtorrent#440]
```
npm WARN engine windows-no-runnable@0.0.6: wanted: {"node":"0.6"} (current: {"node":"4.1.0","npm":"2.14.3"})
npm WARN engine mdns-js@0.4.0: wanted: {"node":"^0.10.3 || ^0.12.0"} (current: {"node":"4.1.0","npm":"2.14.3"})
npm WARN engine xmlbuilder@2.2.1: wanted: {"node":"0.8.x || 0.10.x"} (current: {"node":"4.1.0","npm":"2.14.3"})

> dtrace-provider@0.5.0 install /Users/feross/code/webtorrent/node_modules/chromecasts/node_modules/node-ssdp/node_modules/bunyan/node_modules/dtrace-provider
> node scripts/install.js

---------------
Building dtrace-provider failed with exit code 1 and signal 0
re-run install with environment variable V set to see the build output
---------------

npm WARN deprecated deflate-crc32-stream@0.1.2: module has been merged into crc32-stream
```

**Comment by ericwooley** [comment]
Now that npm3 is out, a lot of warnings are probably forthcoming. There are some changes, especially with peer dependencies that a lot of packages will have to start dealing with, leading to more deprecations etc... Trying to fix them all now is probably going to be a lot like playing wackamole

**Comment by feross** [comment]
Yeah, true. But a lot of these can be fixed with quick PRs to a few different repos. Might as well get the ball rolling on these warnings. npm3 might create a few more, but these ones aren't likely to go away :)

**Comment by gillesdemey** [comment]
Some relevant links to PR's and issues at the appropriate repos:

deflate-crc32-stream in istanbul-middleware
https://github.com/gotwarlost/istanbul-middleware/pull/20

windows-no-runnable
https://github.com/Benvie/node-Windows/issues/10

xmlbuilder2 in node-plist
https://github.com/TooTallNate/plist.js/pull/64

mdns-js in airplay-js, we might want to use [https://www.npmjs.com/package/airplay2](https://www.npmjs.com/package/airplay2) or [https://github.com/feross/node-airplay-js](https://github.com/feross/node-airplay-js) instead?

**Comment by devTristan** [comment]
LIKE NO ONE EVER WAS

**Comment by feross** [comment]
TO CATCH THEM IS MY REAL TEST

**Comment by thebeautyisinthetext** [comment]
TO TRAIN THEM IS MY CAUSE

**Comment by rileyjshaw** [comment]
I WILL TRAVEL, ACROSS THE LAND

**Comment by jimmywarting** [comment]
Is windows-no-runnable so important? I'm not particular a fan of loading a windows package when i'm on a unix environment or dosen't have VLC installed. Could some post-install process work? or better yet solve the one and only thing windows-no-runnable is needed for: [cmd.js#L392](https://github.com/feross/webtorrent/blob/8f7c9285bc4cc599bb625cb79fbd98c483b56db3/bin/cmd.js#L392) without and dependency?

**Comment by feross** [comment]
More warnings to work on removing:

```
npm WARN engine mdns-js@0.4.0: wanted: {"node":"^0.10.3 || ^0.12.0"} (current: {"node":"5.0.0","npm":"3.3.6"})
```

Issue opened on mdns-js: https://github.com/kmpm/node-mdns-js/issues/50

**Comment by whitef0x0** [comment]
@feross Emailed the maintainer of kmpm/node-mdns-js, as it seems he hasn't been maintaining his project in the past few months (with 9 opened issues and 5 unresolved PRs).

**Comment by feross** [comment]
This is the only remaining warning. `windows-no-runnable` is basically abandoned so I want to try to find another package that's small and can replace it.

```
npm WARN engine windows-no-runnable@0.0.6: wanted: {"node":"0.6"} (current: {"node":"4.2.3","npm":"2.14.7"})
```

**Comment by feross** [comment]
Replaced `windows-no-runnable` with [`winreg`](https://www.npmjs.com/package/winreg).

**Comment by feross** [comment]
Released 0.72.0.
